pycsw/core/config.py

```diff
@@ -170,6 +173,7 @@ def __init__(self, prefix='csw30'):
                 'pycsw:Creator': 'creator',
                 'pycsw:Publisher': 'publisher',
                 'pycsw:Contributor': 'contributor',
+                'pycsw:Funder': 'funder',
```

Comment:
Same here, we need to validate this queryable (see other comments in this review).

Reply:
@tomkralidis We are under some pressure to recognise sources of funding in all our metadata. This helps us justify to our funders that their money does have valuable and tangible outcomes. But it is your call in the end.

Reply:
@vjf in discussion with @kalxas, we do not store role codes as queryables currently.  We are fine with adding vertical extents.